packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -351,16 +344,10 @@ contract SortedOracles is ISortedOracles, ICeloVersionedContract, Ownable, Initi
   function medianRate(address token) external view returns (uint256, uint256) {
     EquivalentToken storage equivalentToken = equivalentTokens[token];
     if (equivalentToken.token != address(0)) {
-      (uint256 equivalentMedianRate, uint256 numRates) = medianRateWithoutEquivalentMapping(
+      (uint256 equivalentMedianRate, uint256 _numRates) = medianRateWithoutEquivalentMapping(
```

Comment:
`_numRates` => `denominator`. And looks like this is incorrectly named in several other places (e.g. the `medianRate` function).

Reply:
Fixed, thanks